docs/process/release.md

```diff
@@ -112,6 +112,9 @@ As commits are cherry-picked when PRs are merged, creating the release should be
 ```bash
 git commit -a -m 'release: vXX'
 git tag 'vXX'
+
+# Make sure to run these commands together, as missing tags can cause CI
+# failures.
 git push upstream && git push upstream --tags
```

Comment:
Maybe better `git push upstream --follow-tags`?

Reply:
Interesting, I wasn't aware of that flag. However it seems to only push *annotated* tags, which doesn't fit our current convention of `git tag 'vXXX'`. (Unless we want to switch to using annotated tags.)

> `--follow-tags` only pushes annotated tags.
> 
> ```
> git tag -a -m "I'm an annotation" <tagname>
> ```
> 
> That would be pushed (as opposed to `git tag <tagname>`, a lightweight tag, which would not be pushed, &lt;snip&gt;)

https://stackoverflow.com/a/3745250

Another answer suggests:

```
git push upstream tag vXXX
```

But I think that is not well-understood behavior and is probably less intuitive than `git push upstream && git push upstream --tags`.

https://stackoverflow.com/a/55882829